pkg/config/config.go

```diff
@@ -1067,6 +1071,17 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 			return fmt.Errorf("initialize nsmgr: %w", err)
 		}
 
+		if c.EnableCriuSupport {
+			if err := validateCriuInPath(); err != nil {
+				c.EnableCriuSupport = false
+				return errors.New("cannot enable checkpoint/restore support without the criu binary in $PATH")
+			} else {
```

Comment:
nit: don't need this else because we return in the previous block

Reply:
Right. Fixed.